specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -85,15 +98,15 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/enable": {
-      "post": {
-        "operationId": "Subscriptions_Enable",
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/SupportPlan/default": {
```

Comment:
Just to confirm - can a subscription have multiple support plans, or just one (named default)?